docs/installation.md

```diff
@@ -138,6 +138,23 @@ extraWhitelistPaths:
   - '/ext/faster'
 ```
 
+### Configure additional v1/statement-like paths
+
+The Trino client protocol specifies that queries are initiated by a POST to `v1/statement`. 
```

Comment:
```suggestion
The Trino client protocol specifies that queries are initiated by a POST to `/v1/statement`. 
```
Might be good to say that path should start with `/` as it will give an error anyway